Cilsil/Log.cs

```diff
@@ -52,10 +75,10 @@ public static void RecordUnknownInstruction(string instruction)
         /// </summary>
         public static void PrintAllUnknownInstruction()
```

Comment:
I have modified this so that unknown instruction list is only printed in Debug environment; is that the desired behavior?